pom.xml

```diff
@@ -143,6 +147,12 @@ THE SOFTWARE.
       <artifactId>powermock-api-mockito</artifactId>
       <version>1.5</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.mockito</groupId>
+          <artifactId>mockito-all</artifactId>
```

Comment:
It is possible a newer version of `powermock-api-mockito` fixes their dependencies; I did not want to upgrade PowerMock here.

Anyone publishing `*-all` artifacts on Central with `jar` packaging should be given a stern lecture…